Eatery/Controllers/Eateries/EateriesMenuViewController.swift

```diff
@@ -136,12 +137,18 @@ class EateriesMenuViewController: ImageParallaxScrollViewController {
             self?.view.layoutIfNeeded()
         }
     }
-
     func createHeroModifiers(_ groups: HeroModifierGroups...) -> [HeroModifier] {
         [.useGlobalCoordinateSpace, .whenPresenting(.delay(0.15))]
             + (groups.contains(.fade) ? [.fade] : [])
             + (groups.contains(.translate) ? [.translate(y: 32), .timingFunction(.deceleration)] : [])
     }
+    // 0.15 is default delay used across all presenting hero animations
```

Comment:
Newline before. You can make this a documentation comment with `///` instead of `//`